icc-x-api/icc-patient-x-api.ts

```diff
@@ -1179,6 +1590,133 @@ export class IccPatientXApi extends IccPatientApi {
     })
   }
 
+  exportUsingPost(user: models.User, patId: string, ownerId: string): Promise<{ id: string }> {
```

Comment:
Except some methods, the content of the exportUsingPost is the same than the export method.
Please reuse the common code and avoid duplication